internal/platform/writer/console.go

```diff
@@ -21,13 +21,32 @@ var _ scan.Writer = Console{}
 // Console is a [scan.Writer] implementation that writes the output
 // to the given [io.Writer], following console/terminal standards
 // in a human-friendly fashion.
+
+type ConsoleOption func(*Console)
+
 type Console struct {
-	writer io.Writer
+	writer     io.Writer
+	pocEnabled bool
+}
+
+func WithPOCEnabled(enabled bool) func(*Console) {
```

Comment:
Can you write a short doc about what's the expected behavior once enabled, what will be printed (or not) for each type of "event" (match, error, etc), please?